courses/machine_learning/deepdive/03_tensorflow/labs/d_traineval.ipynb

```diff
@@ -9,12 +9,30 @@
    "source": [
```

Comment:
<p>Same comment regarding the Monitor Training.  If TensorBoard no longer being used, no need for the "Monitor training" section or the code:  OUTDIR = './taxi_trained'.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/955/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
I agree with Gwendolyn's comment.